app/presenters/search_presenter.rb

```diff
@@ -11,7 +11,11 @@ def to_json
       description: document.summary,
       link: document.base_path,
       indexable_content: indexable_content,
-      organisations: organisation_slugs,
+      # FIXME: This line was dependent on a method called #organisation_slugs,
+      # which in turn used publishing_api.get_content to map org content_ids to
+      # organisation slugs. It was stubbed in many places in spec/, but that method
+      # call always returned a blank list in the real world.
+      organisations: [],
```

Comment:
I guess I meant "what would the impact be if this were to go live without the Finding Things work?" In theoretical terms I'm curious about what purpose the inflamed appendix we just took out was *for*, but in practical terms I suppose I should just be happy the patient isn't dead. To use an uncomfortably *medical* metaphor...